docs/en/fine_tuning/how_to/creatings.mdx

```diff
@@ -0,0 +1,125 @@
+---
+weight: 30
+---
+
+# Creating Fine-Tuning Tasks
+
+## Preparation
+
+### Datasets
+
+We support reading datasets from your S3 storage as well as from internal datasets.
+If you want to use S3 storage, note that you need to add the following annotations to the corresponding Secret `s3-url`, `s3-name`, `s3-path`.
```

Comment:
If you are using S3 storage, you need to create a `Secret` under your namespace like below: